src/azure-cli/azure/cli/command_modules/acr/repository.py

```diff
@@ -284,6 +285,10 @@ def acr_repository_update(cmd,
         json_payload.update({
             'writeEnabled': write_enabled
         })
+    if teleport_enabled is not None:
+        json_payload.update({
+            'teleportEnabled': teleport_enabled
```

Comment:
is the value True false? Not enabled disabled?

Reply:
can you add this to an example in testing or add a new command to test it?
You need to regenerate the records after it.

Reply:
It's three state, value can be `true`, `false` or `null`

Reply:
What testing? is it unit tests? 
We are going to add integrations tests using our CIs

Reply:
I meant for the CLI. There is a test folder in ACR this is run when you try to submit changes. But as you say probably the CI are enough.